artifact.json

```diff
@@ -3,8 +3,8 @@
   "vcsURL": "https://github.com/guardian/cloudwatch-logs-management",
   "actions": [
     {
-      "action": "cloudwatch-logs-management-cfn",
-      "path": "template.yaml",
+      "action": "cdk.out",
```

Comment:
All the synthed templates sit here, so instruct `@guardian/node-riffraff-artifact` to upload the entire directory.